src/types/annotator.ts

```diff
@@ -376,4 +376,4 @@ export type SideBySideMode = SideBySideOptions['mode'];
  * - "selection" - Use the current text or DOM selection
  * - "rect" - Draw a rectangle to select a region of the document
  */
-export type AnnotationTool = 'selection' | 'rect';
+export type AnnotationTool = 'selection' | 'rect' | 'point';
```

Comment:
This is called "point" here for contrast with "rect", even though in the UI we use the term/icon "pin".